app/src/main/assets/about-es.html

```diff
@@ -12,13 +12,13 @@
 
 <p>Creado por <i>Jeroen Carolus</i></p>
 
-<p>Para el apoyo , la informaci�n acerca de las actualizaciones y la versi�n m�s reciente : por favor vaya a la p�gina web de desarrolladores <a href='http://www.jwtc.nl/'>jwtc.nl</a>.</p>
+<p>Para el apoyo , la informaci�n acerca de las actualizaciones y la versi�n m�s reciente : por favor vaya a la p�gina web de desarrolladores <a href='http://www.jwtc.nl/'>jwtc.nl</a>.</p>
```

Comment:
Can you check if the encoding of the files is correct (UTF-8), because this diff looks like something may be wrong here. Maybe opened in another (HTML) editor?